<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCatchBinDialog</class>
 <widget class="QDialog" name="QCatchBinDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>file:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="filePath"/>
       </item>
       <item>
        <widget class="QPushButton" name="browse">
         <property name="text">
          <string>browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>start:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="start"/>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>end:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="end"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>format:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="outPutFileName"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QPushButton" name="catchButton">
         <property name="text">
          <string>Catch</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quit">
         <property name="text">
          <string>Quit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
